lib/puppet.rb

```diff
@@ -74,7 +74,7 @@ class << self
       end
 
       if locale_path
-        if Gem.loaded_specs['gettext-setup'].version < Gem::Version.new('0.8')
+        if Gem::Version.new(GettextSetup::VERSION) < Gem::Version.new('0.8')
```

Comment:
Yes, in the context of Puppet within puppet-server. They vendor their own gems, and the older versions of it (e.g. less than 2.7.2, I think) have something like gettext-setup 0.6.